doc/tutorials/setup_logical_partition_scenario_on_kbl_nuc_i7.rst

```diff
@@ -0,0 +1,154 @@
+.. _setup_logical_partition_scenario_on_kbl_nuc_i7:
+
+Guidelines of Setup Logical Partition Scenario on KBL NUC i7
+############################################################
+
+ACRN hypervisor supports logic partition scenario, in which the User OS (such as Clear Linux) running in a pre-launched VM can bypass the ACRN hypervisor and directly access isolated PCI devices. The following guidelines provide step by step instructions on how to set up the ACRN hypervisor logical partition scenario on KBL NUC i7 boards running two pre-launched VMs. The logical partition scenario on KBL NUC i7 is shown in :numref:`logical_partition_scenario_on_kbl_nuc_i7`.
+
+.. figure:: images/logical_partition_scenario_on_kbl_nuc_i7.png
+   :align: center
+   :width: 400px
+   :name: logical_partition_scenario_on_kbl_nuc_i7
+
+   The logical partition scenario on KBL NUC i7
+
+Prerequisites
+*************
+
+Disable "Intel Hyper Threading Technology" in the BIOS.
```

Comment:
Can we add a short explanation as to why we make this recommendation (or is strictly required)?

Reply:
@gvancuts There are two reasons: one is to mitigate L1TF, other is to avoid interference on the logical partition scenario between logical cores.

Reply:
add doc v2 version